tests/e2e/specs/media/insertMediaFromLibrary.js

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { createNewStory } from '../../utils';
+
+describe('Inserting Media from Media Library', () => {
+  it('should insert an image by clicking on it', async () => {
+    await createNewStory();
+
+    // Clicking will only act on the first element.
+    await expect(page).toClick('[data-testid="mediaElement"]');
+
+    // Matches the element's frame layer,
+    // using it as an indicator for successful insertion.
+    await expect(page).toMatchElement('[data-testid="FramesLayer"]');
```

Comment:
That's the whole layer (container), not a specific element. And the whole layer is always there.